app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -239,6 +268,38 @@ class OnboardingFragmentTest {
     }
   }
 
+  @Test
+  fun testOnboardingFragment_Slide1_hasCorrectContentDescription() {
```

Comment:
As per my earlier comment, we use camelCase format in naming test functions. 

```suggestion
  fun testOnboardingFragment_slide1_hasCorrectContentDescription() {
```

Reply:
Done